spec/integration/admin/manager_approves_admins_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Admin do
+  it 'can not approve an admin without a login' do
+    admin = Fabricate(:admin, status: 0)
+
+    post admin_enable_admin_dashboard_path(admin)
+
+    expect(response).to redirect_to(new_admin_session_path)
+  end
+
+  it 'can not approve an admin without a login' do
+    admin = Fabricate(:admin, status: 0)
+    another_admin = Fabricate(:admin, status: 1)
+
+    login_as another_admin, scope: :admin
+
+    post admin_enable_admin_dashboard_path(admin)
+
+    expect(Admin.active).to eq([another_admin])
```

Comment:
esse teste pode ficar um pouco mais sucinto usando o matcher de change se quiserem dar uma olhada, ficaria algo assim:

```suggestion
    admin = Fabricate(:admin, status: :blocked)
    another_admin = Fabricate(:admin, status: :active)

    login_as another_admin, scope: :admin
    expect { post admin_enable_admin_dashboard_path(admin) }
      .not_to change { Admin.active.count }
```